code/ssh2userauth.c

```diff
@@ -972,7 +972,7 @@ static void ssh2_userauth_process_queue(PacketProtocolLayer *ppl)
 					{
 						DWORD siglen = 0;
 						ptrlen datatosign = ptrlen_from_strbuf(sigdata);
-						LPBYTE sig = cert_sign(key, datatosign.ptr, datatosign.len, &siglen, hwnd);
+						LPBYTE sig = cert_sign(key, datatosign.ptr, datatosign.len, &siglen, hwnd, 0);
```

Comment:
Did you test without Pageant?  I believe this is the call that would be used to sign without Pageant.

Reply:
Yes, I tested at least CAPI via putty directly without pageant running/configured for use.  I think I may have tested a ppk as well but I don't immediately recall.  I can do so later though.